### PR TITLE
feat: replace rest calls with rpc on man and otp verification services

### PR DESCRIPTION
**Pull request by progsmile**
#4464

### CODE REVIEW

File: apps/core/lib/core/declaration_requests/api/v1/creator.ex [hunk 1]
```diff
@@ -798,8 +798,8 @@ defmodule Core.DeclarationRequests.API.V1.Creator do
       {:ok, printout_content} ->
         put_change(changeset, :printout_content, printout_content)
 
-      {:error, error_response} ->
-        add_error(changeset, :printout_content, format_error_response("MAN", error_response))
+      {:error, _} ->
+        add_error(changeset, :printout_content, format_error_response("MAN", "Remote server internal error"))
```
Comment (imax-iva) [inline apps/core/lib/core/declaration_requests/api/v1/creator.ex]: may need to raise Exception?

File: apps/core/lib/core/man/templates/contract_request_printout_form.ex [hunk 2]
```diff
@@ -29,10 +29,9 @@ defmodule Core.Man.Templates.CapitationContractRequestPrintoutForm do
 
   def render(
         %CapitationContractRequest{parent_contract_id: parent_contract_id, contract_number: contract_number} =
-          contract_request,
-        headers
+          contract_request
       )
-      when not is_nil(parent_contract_id) do
+      when parent_contract_id != nil do
```
Comment (imax-iva) [inline apps/core/lib/core/man/templates/contract_request_printout_form.ex]: why? :)

File: apps/core/lib/core/declaration_requests/api/v1/creator.ex (hunk 1 above)
Comment (progsmile) [reply apps/core/lib/core/declaration_requests/api/v1/creator.ex]: support old behaviour

File: apps/core/lib/core/man/templates/contract_request_printout_form.ex (hunk 2 above)
Comment (progsmile) [reply apps/core/lib/core/man/templates/contract_request_printout_form.ex]: suppose formatter format in single line

**Pull request merged**